File: Python/2108.py
# 2108. Find First Palindromic String in the Array
# https://leetcode.com/problems/find-first-palindromic-string-in-the-array/

class Solution:
  def firstPalindrome(self, words: List[str]) -> str:
    out = ""

    for w in words:
      if (w == w[::-1]):
        out = w
        break

    return out

File: Rust/2108.rs
/**
 * 2108. Find First Palindromic String in the Array
 * https://leetcode.com/problems/find-first-palindromic-string-in-the-array/
 */

impl Solution {
  pub fn first_palindrome(words: Vec<String>) -> String {
    let mut out: String = "".to_string();

    for w in words {
      let _w: String = w.chars().rev().collect::<String>();

      if w == _w {
        out = w;
        break;
      }
    }

    return out;
  }
}
